build.mjs

```diff
@@ -96,10 +109,19 @@ const esbuildSassPlugin = {
   },
 };
 
-const renderTemplate = async (template) => {
+const renderTemplate = async (template, language) => {
+  const localePath = path.join(__dirname, "src", "locales", language + '.json');
+  const t = JSON.parse(await readFileAsync(localePath));
```

Comment:
Sorry for the unclear feedback from earlier. `build.mjs` already does `import fs from "node:fs/promises"` which provides promisified filesystem APIs.

```suggestion
  const t = JSON.parse(await fs.readFile(localePath));
```